sqlglot/dialects/oracle.py

```diff
@@ -30,6 +30,11 @@ def _build_timetostr_or_tochar(args: t.List) -> exp.TimeToStr | exp.ToChar:
 
     return exp.ToChar.from_arg_list(args)
 
+def _build_addmonths(args: t.List) -> exp.AddMonths:
+    return exp.AddMonths(
+        this=seq_get(args, 0),
+        integer=seq_get(args, 1)
+    )
```

Comment:
This is unnecessary, `ADD_MONTHS` gets automatically parsed as `AddMonths` due to its name.
```suggestion
def _build_addmonths(args: t.List) -> exp.AddMonths:
    return exp.AddMonths(
        this=seq_get(args, 0),
        integer=seq_get(args, 1)
    )
```